graphx/src/main/scala/org/apache/spark/graphx/util/PeriodicGraphCheckpointer.scala

```diff
@@ -87,10 +87,10 @@ private[mllib] class PeriodicGraphCheckpointer[VD, ED](
 
   override protected def persist(data: Graph[VD, ED]): Unit = {
     if (data.vertices.getStorageLevel == StorageLevel.NONE) {
-      data.vertices.persist()
```

Comment:
isn't persist better? this could potentially support different storage level later